java/core/src/java/org/apache/orc/impl/DynamicIntArray.java

```diff
@@ -137,8 +135,8 @@ public String toString() {
     return sb.append('}').toString();
   }
 
-  public int getSizeInBytes() {
-    return 4 * initializedChunks * chunkSize;
+  public long getSizeInBytes() {
```

Comment:
From this PR, I believe this is the valid one.

Reply:
Sure